templates/taxi/car_list.html

```diff
@@ -1,12 +1,13 @@
 {% extends "base.html" %}
 
 {% block content %}
-  <h1>Car list</h1>
+  <h1>Car list <a style="float: right" href="{% url 'taxi:car-create'%}">+</a></h1>
+  
   {% if car_list %}
     <ul>
       {% for car in car_list %}
         <li>
-          <a href="{% url "taxi:car-detail" pk=car.id %} ">{{ car.id }}</a>
+          <a href="{% url "taxi:car-detail" pk=car.id %}">{{ car.id }}</a>
```

Comment:
The `url` template tag uses double quotes, which is inconsistent with the rest of the template. It's recommended to use single quotes for consistency: `{% url 'taxi:car-detail' pk=car.id %}`.